api/inventorypb/agents.proto

```diff
@@ -34,6 +34,10 @@ enum AgentType {
 message PMMAgent {
   // Unique randomly generated instance identifier.
   string agent_id = 1;
+  // Login represent login for vmagent auth config.
```

Comment:
let's move these fields into register node response and leave only one field: token